Features/Context/PlatformUI.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+/**
+ * File containing the main context class for PlatformUI
+ * This flie contains the mapping of the BDD sentences to the functions that implement them
+ * Also contains the necessary initializations
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\PlatformUIBundle\Features\Context;
+
+use EzSystems\BehatBundle\Context\Browser\Context;
+use Behat\Gherkin\Node\TableNode;
+use PHPUnit_Framework_Assert as Assertion;
+
+class PlatformUI extends Context
+{
+    const USER = "admin";
```

Comment:
We should make that configurable somehow. It is quite convenient to run BDD tests on an existing instance without changing your login & password.